### ISSUE TITLE
Open and close sidebar

### ISSUE THREAD

**Issue by betsy-chou** [phosphor-icons/homepage#518]
**Describe the requested icon(s)**
I'd like an icon that serves as a button for opening and closing a sidebar. There is a currently two icons in Phosphor: sidebar and detailed sidebar, but I don't think they're clear enough as an affordance. 

**References**
The one I'm currently using is from Material Design where there's an arrow inside to indicate the direction in which the sidebar will move. 
<img width="88" alt="Screenshot 2024-06-24 at 5 31 52 PM" src="https://github.com/phosphor-icons/homepage/assets/161088607/185eb470-3e83-4b0d-bb0b-d02a4f63cdbd">

I've also seen other alternatives where the sidebar is filled when it's open, but I don't think those are very clear.

**Comment by patfromcloudeats** [comment]
I would like to second this. Here are references from Confluence / Jira:

<img width="290" height="137" alt="Image" src="https://github.com/user-attachments/assets/c58e67c9-3154-469d-b795-8fb5b3a73494" />
<img width="294" height="146" alt="Image" src="https://github.com/user-attachments/assets/8cd5235f-9495-4093-a781-6aa5ff8edb7a" />

**Comment by minoraxis** [comment]
Good call, these can be hard to read. We'll add arrow variants to the backlog and see if there are any other solutions to explore
